CIScripts/BuildFunctions.ps1

```diff
@@ -0,0 +1,151 @@
+class Repo {
+    [string] $Url;
+    [string] $Branch;
+    [string] $DefaultBranch;
+    [string] $Dir;
+
+    Repo ([string] $Url, [string] $Branch, [string] $Dir, [string] $DefaultBranch) {
```

Comment:
I think changing the order of parameters here to be consistent with the order of class fields would make the code easier to read. Or the other way around.